apps/earthbound/pages/_app.tsx

```
@@ -0,0 +1,151 @@
+
+export function getLibrary(provider: ExternalProvider | JsonRpcFetchFunc) {
+  const library = new Web3Provider(provider);
+  return library;
+}
+
+const prepareDefaultConfig = () => {
+  const general =
+    configDescription.find((config) => config.slug === 'general')?.value ?? {};
+  const transformedObject = Object.entries(general).reduce(
+    (acc, [key, value]) => {
+      return { ...acc, [key]: '' };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)